WORKFLOWS.md

```diff
@@ -0,0 +1,25 @@
+# Reusable Workflows
+
+In addition to GitHub community health templates, this repository defines reusable workflows for the organization.
+
+## Continuous Integration
+
+To use the CI workflow:
+
+```yaml
+# In a package repository, in the file .github/workflows/continuous-integration.yml:
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+      - 'renovate/*'
+    tags:
+
+jobs:
+  ci:
+    uses: laminas/.github/workflows/continuous-integration.yml@ci-v1
```

Comment:
I'm just curious why the `ci-` prefix, do you expect to have different v1 tags for this or is it a best practice?

Reply:
Basically, so we can have multiple workflows in here (e.g., one for docs, one for release), and create tags for changes in each.

So, for example, if we made a change to the releases workflow that prompts a new major, that new major would not apply to the other two workflows.

The other option, of course, is to do these in separate branches; using a branch ref would allow them to change over time to pick up fixes or features.

I'm really not sure what would be the best solution, to be honest. Most of the time, I don't anticipate these changing very often, but when they do, we want to allow consumers to opt-in to the changes, instead of forcing users to opt-out by updating to specify an old ref.

Reply:
Ah I see now. In my mind using different files would have been all you need, but I didn't think that through properly.

Reply:
Updated!